### ISSUE TITLE
Test set for imbalanced dataset.

### ISSUE THREAD

**Issue by mongoose54** [NVIDIA/DIGITS#281]
Using DIGITS I am trying to build a network to classify a 2-class dataset. Class A has ~170K images and Class B has ~20K. The setup is as follows: 65% for training, 25% for validation and 10% for testing. When I ran modelling I get ~90% accuracy. While the accuracy is quite high I have serious doubts that is true. The reason is that the data set is quite imbalanced. It would not be that of a problem if the test set is balanced (i.e. 50% from Class A and 50% from Class B). So my question is how does DIGITS create the test set? Is it balanced or not regardless of the dataset? Thanks!

**Comment by lukeyeager** [comment]
If you use the "split folder" feature of DIGITS, it will take 25% of the images **in each category** for the validation set. If you want to do something else, you'll have to create your own textfiles and upload them to DIGITS. The tool at `tools/parse_folder.py` may be helpful to create the initial textfiles, and then you can split them up as you like.

The accuracy in this case is "true", but misleading.
